sdk/identity/azure-identity/src/test/java/com/azure/identity/implementation/MSITokenTests.java

```diff
@@ -38,21 +39,34 @@ public void canParseLong() {
     public void canDeserialize() {
         String json = "{\n"
             + "  \"access_token\": \"fake_token\",\n"
-            + "  \"refresh_token\": \"\",\n"
             + "  \"expires_in\": \"3599\",\n"
-            + "  \"expires_on\": \"1506484173\",\n"
-            + "  \"not_before\": \"1506480273\",\n"
-            + "  \"resource\": \"https://managementazurecom/\",\n"
```

Comment:
why are we removing all this json from test ?

Reply:
It's extraneous json that isn't deserialized by the type anyway. The only serialized fields on an `MSIToken` are `access_token`, `expires_in`, and `expires_on`